pwiz_tools/Skyline/SkylineGraphs.cs

```diff
@@ -908,6 +908,35 @@ private void charge4MenuItem_Click(object sender, EventArgs e)
             _graphSpectrumSettings.ShowCharge4 = !_graphSpectrumSettings.ShowCharge4;
         }
 
+        private void SynchMzScaleToolStripMenuItemClick(object sender)
+        {
+            if (_graphFullScan == null || !_graphFullScan.Visible
+                                       || _graphSpectrum == null || !_graphSpectrum.Visible)
+                return;
+            Settings.Default.SyncMZScale = synchMzScaleToolStripMenuItem.Checked;
+            //testing support
+            var source = sender as IMzScaleCopyable ??
+                         (IMzScaleCopyable)(synchMzScaleToolStripMenuItem.Owner as ContextMenuStrip)?.SourceControl?.Parent?.Parent;

```

Comment:
I think you should do:
(IMzScaleCopyable)(synchMzScaleToolStripMenuItem.Owner as ContextMenuStrip)?.SourceControl?.FindForm()

The problem with "Parent?.Parent" is that it would stop working if someone redesigned the form so that the ZedGraphControl was inside of an extra Panel or something.